Sources/Controllers/2.0/OAEditFavoriteViewController.mm

```diff
@@ -203,6 +211,15 @@ - (void) commonInit
     _ininialName = self.name;
     _ininialGroupName = self.groupTitle;
     _editingTextFieldKey = @"";
+    
+    _selectCategorySectionIndex = -1;
```

Comment:
If these values are constants you need to define them as such. Do not define constants as private fields.